osquery/distributed/distributed.cpp

```diff
@@ -117,11 +119,56 @@ Status Distributed::serializeResults(std::string& json) {
     doc.add(result.request.id, arr, queries_obj);
     doc.add(result.request.id, result.status.getCode(), statuses_obj);
     doc.add(result.request.id, result.message, messages_obj);
+
+    RecursiveLock lock(performance_mutex_);
+    auto obj = doc.getObject();
+    if (performance_.count(result.request.id) > 0) {
+      auto perf = performance_[result.request.id];
+      obj.AddMember("executions",
+                    static_cast<uint64_t>(perf.executions),
+                    obj.GetAllocator());
+      obj.AddMember("last_executed",
+                    static_cast<uint64_t>(perf.last_executed),
+                    obj.GetAllocator());
```

Comment:
Is `last_executed` meaningful in this context? Will it always be basically "now" for a distributed query?